eng/Versions.Scaffold.AspNet.props

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need to have these (this file and `eng/Versions.Scaffold.Aspire.props` and the existing `eng/Versions.Scaffold.props`) separate immediately since they're identical (or nearly identical in the case of the existing file)? Or can we split them when necessary?

Reply:
These are going to be different pretty quickly I think as their dependencies and properties will diverge. Also since we might develop on these tools separately, its nice to have this separation for clarity and if we need to release them not in sync.